src/redux/settings.ts

```diff
@@ -254,7 +255,7 @@ export const settingsChangeNativeCurrency =
 
 // -- Reducer --------------------------------------------------------------- //
 export const INITIAL_STATE: SettingsState = {
-  accountAddress: '',
+  accountAddress: '' as Address,
```

Comment:
this might be problematic in other areas of the app and tslint might complain to you about it

Reply:
types are fine, what do you think is problematic?

Reply:
okay! just didn't know if something would yell at you around the app